JortPob/Paramanager.cs

```diff
@@ -406,23 +421,35 @@ public WeatherData(string name, List<string> match, int MapInfoParamId, int MapR
             {
                 this.name = name;
                 this.match = match;
-                for (int i = 0; i < match.Count(); i++) { match[i] = match[i].Trim().ToLower(); }
                 this.MapInfoParamId = MapInfoParamId;
                 this.MapRegionParamId = MapRegionParamId;
                 this.rem = rem;
             }
         }
 
         // @TODO: maybe move weatherdata to layout or its own class since its used by multiple toher thingsthingso
+        public static WeatherData GetWeatherData(string region)
+        {
+            foreach(WeatherData wd in EXTERIOR_WEATHER_DATA_LIST)
+            {
+                foreach(string m in wd.match)
+                {
+                    if (m.ToLower().Trim() == region.ToLower().Trim()) { return wd; }
```

Comment:
nit: move the `region.ToLower().Trim()` call outside the `foreach` loops and store it in a variable so we only need to do it once